File: scraper/bbsr.php
<?php 
$host = "http://www.bbsr.bund.de/BBSR/DE/Aktuell/Forschungsprojekte/forschungsprojekte_node.html"; 
$filestring = file_get_contents($host); 
$startpos = 0; 
while($pos = strpos($filestring, "<a href", $startpos)) 
{ 
     $string = substr($filestring, $pos, strpos($filestring, "</a>", $pos + 1) - $pos); 
    if(stristr($string, 'id="ausschreibung"')) { 
        echo $string."</br>"; 
    } 
    $startpos = $pos + 1; 
} 
?>

File: wombat/bbsr.rb
require 'wombat'

class BBSR
  include Wombat::Crawler
  
  base_url "http://www.bbsr.bund.de/"
  path "/BBSR/DE/Aktuell/Forschungsprojekte/forschungsprojekte_node.html"
  Ausschreibungen do |a|
    a.was "css=table#ausschreibung", :list
  end
end

puts BBSR.new.crawl
